src/tests/system/tests/test_sssctl.py

```diff
@@ -545,3 +545,258 @@ def test_sssctl__analyze_with_logdir_and_non_root_privileges(client: Client, pro
+    :expectedresults:
+        1. Command executes successfully
+        2. Output shows correct user details
+    :customerscenario: False
+    """
+    provider.user("user1").add(uid=10001, gid=10001, password="Secret123")
+    client.sssd.start()
+
+    result = client.sssctl.user_show("user1")
+    assert result.rc == 0, f"user-show failed: {result.stderr}"
+    assert "user1" in result.stdout, f"Username not found in output: {result.stdout}"
+
+    user_indicators = ["uid", "gid", "name:", "user1"]
+    details_found = any(indicator in result.stdout.lower() for indicator in user_indicators)
+    assert details_found, f"User details not found in output: {result.stdout}"
+
+
+@pytest.mark.tools
+@pytest.mark.topology(KnownTopologyGroup.AnyProvider)
+def test_sssctl__group_show(client: Client, provider: GenericProvider):
+    """
+    :title: Verify sssctl group-show displays correct group information
+    :setup:
+        1. Add group to Backend
+        2. Start SSSD
+    :steps:
+        1. Run 'sssctl group-show <groupname>'
+        2. Verify group information is displayed
+    :expectedresults:
+        1. Command executes successfully
+        2. Output shows correct group details
+    :customerscenario: False
+    """
+    provider.group("group1").add(gid=20001)
+    client.sssd.start()
+
+    result = client.host.conn.exec(["sssctl", "group-show", "group1"], raise_on_error=False)
+    assert result.rc == 0, f"group-show failed: {result.stderr}"
+    assert "group1" in result.stdout, f"Group name not found in output: {result.stdout}"
+
+    group_indicators = ["gid", "name:", "group1"]
+    details_found = any(indicator in result.stdout.lower() for indicator in group_indicators)
+    assert details_found, f"Group details not found in output: {result.stdout}"
+
+
+@pytest.mark.tools
+@pytest.mark.topology(KnownTopologyGroup.AnyProvider)
+def test_sssctl__netgroup_show(client: Client, provider: GenericProvider):
+    """
+    :title: Verify sssctl netgroup-show displays correct netgroup information
+    :setup:
+        1. Configure netgroup in Backend
+        2. Start SSSD
+    :steps:
+        1. Run 'sssctl netgroup-show <netgroupname>'
+        2. Verify netgroup information is displayed
+    :expectedresults:
+        1. Command executes successfully
+        2. Output shows correct netgroup details
+    :customerscenario: False
+    """
+    provider.netgroup("ng1").add()
+    client.sssd.start()
+
+    result = client.host.conn.exec(["sssctl", "netgroup-show", "ng1"], raise_on_error=False)
+    assert result.rc == 0, f"netgroup-show failed: {result.stderr}"
+    assert "ng1" in result.stdout, f"Netgroup name not found in output: {result.stdout}"
+
+    netgroup_indicators = ["netgroup", "member", "ng1", "triple"]
+    details_found = any(indicator in result.stdout.lower() for indicator in netgroup_indicators)
+    assert details_found or result.rc == 0, f"Netgroup details not found in output: {result.stdout}"
```

Comment:
You already checked for `result.rc == 0` before, so there's no need to check it again